module/unit.sh

```diff
@@ -243,10 +243,10 @@ function testCoverage() {
                 if [ $count -gt 0 ]; then
                     local -a fns=(
                         $(grep -oE "${regex}" ${modulepath} |
-                            sed -e "s/^function :\{0,2\}${module}:\([^.()]\+\)\(\.[a-z]\+\)\?()/\1/"
+                            sed -re "s/^function :{0,2}${module}:([^.()]+)(\.([a-z]+))?\(\)/\1\u\3/"
                         )
                     )
-                    for fn in ${fns[@]}; do
+                    for fn in "${fns[@]}"; do
```

Comment:
So that each token is also quoted, for example:

```bash
let -a foo=(
  "a a"
  "b b b"
  "c c c c"
)
```

Without the quote, the other end would read 9 tokens, rather than 3.